app/views/content/life-as-a-teacher/teaching-as-a-career/abigails-career-progression-story/_article.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+      <h2>Becoming a maths teacher</h2>
+      <p>I always enjoyed maths at school and had the opportunity to take my maths GCSE early. I went on to study Maths and Management at university, always with the idea that I would go into a leadership or management role, but with no certainty of what industry I would enter. </p>
+      <p>A few years after I finished university, my old school sent out a letter to former students advertising their teacher training course. Around the same time, I went to meet one of my friends who was working at a [Get Into Teaching event](/events) and got chatting about a career in teaching. It seemed like a great opportunity!</p>
```

Comment:
event link needs reformatting!

![image](https://github.com/user-attachments/assets/a47cd63d-4c6b-4c4b-9eca-8ce74747a218)

Reply:
Also - link should go to `/events/about-get-into-teaching-events`